.github/workflows/aqa.yml

```diff
@@ -22,16 +22,16 @@ jobs:
     steps:
 
     - name: Run AQA Tests - ${{ matrix.target }}.${{ matrix.suite }}
-      uses: adoptium/run-aqa@v2
-      with: 
+      uses: adoptium/run-aqa@v2.0.1
+      with:
         jdksource: 'customized'
-        customizedSdkUrl: ${{ env.CUSTOMIZED_SDK_URL }} 
-        aqa-testsRepo: 'adoptium/aqa-tests:v0.9.1-release' # Make sure this branch is set to the latest release branch
+        customizedSdkUrl: ${{ env.CUSTOMIZED_SDK_URL }}
+        aqa-testsRepo: 'adoptium/aqa-tests:v0.9.6-release' # Make sure this branch is set to the latest release branch
```

Comment:
Maybe another parameter sent in? I suspect this will change with every single release...

Reply:
Parametrized though set a default for the current version.